build/get-webpack-config.js

```diff
@@ -465,17 +470,20 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
       // in dev because the CLI will not exit with an error code if the option is enabled,
       // so failed builds would look like successful ones.
       watch && new webpack.NoEmitOnErrorsPlugin(),
-      new InstrumentedImportDependencyTemplatePlugin(
-        runtime !== 'client'
-          ? // Server
+      runtime === 'server'
+        ? // Server
+          new InstrumentedImportDependencyTemplatePlugin(
             state.mergedClientChunkMetadata
-          : /**
-             * Client
-             * Don't wait for the client manifest on the client.
-             * The underlying plugin handles client instrumentation on its own.
-             */
-            void 0
-      ),
+          )
+        : /**
+           * Client
+           * Don't wait for the client manifest on the client.
+           * The underlying plugin handles client instrumentation on its own.
```

Comment:
This is no longer accurate now that the i18n manifest comes from outside.
```suggestion
           * The underlying plugin is able determine client chunk metadata on its own.
```